reflex/components/el/elements/media.pyi

```diff
@@ -1633,9 +1633,9 @@ class Line(BaseHTML):
         Args:
             *children: The children of the component.
             x1: The x-axis coordinate of the line starting point.
-            x2: The x-axis coordinate of the line ending point.
+            x2: The x-axis coordinate of the the line ending point.
```

Comment:
when i ran `poetry run python scripts/make_pyi.py`, this happened, not sure why it added an extra 'the'